lib/bolt_server/acl.rb

```diff
@@ -0,0 +1,39 @@
+# frozen_string_literal: true
```

Comment:
git-- this diff is so annoying. I'm assuming you just moved these files?

Reply:
acl just moved

config changed a bit to fix some bugs and make it easier to add new keys.
the app changes are small.